ydb/core/fq/libs/row_dispatcher/format_handler/parsers/json_parser.cpp

```diff
@@ -425,24 +442,112 @@ class TJsonParser : public TTopicParserBase {
     }
 
     const TVector<ui64>& GetOffsets() const override {
-        return Buffer.Offsets;
+        return !ParsingFailedRowCount ? Buffer.Offsets : OutputOffsets;
     }
 
     TValueStatus<std::span<NYql::NUdf::TUnboxedValue>> GetParsedColumn(ui64 columnId) override {
-        if (auto status = Columns[columnId].GetStatus(); status.IsFail()) {
+        if (auto status = Columns[columnId].GetStatus(); !Config.SkipErrors && status.IsFail()) {
             return status;
         }
         return ParsedValues[columnId];
     }
 
 protected:
+
     TStatus DoParsing() override {
```

Comment:
Кажется лишняя строка


```suggestion
protected:
    TStatus DoParsing() override {
```